homework 08_11/Task8.java

```diff
@@ -0,0 +1,76 @@
+import static java.lang.Math.*;
+import java.util.Date;
+import java.util.Calendar;
+import java.util.Scanner;
+
+class Task8 {
+public static void main(String[] args) {
+Scanner myObj = new Scanner(System.in);
+System.out.println("Please enter the year:");
+int year = myObj.nextInt();
+System.out.println(" ");
+if (year<100){
+System.out.println("This is I century");
+
+}else if(year>=100 && year<200){
+System.out.println("This is II century");
+
+}else if(year>=200 && year<300){
+System.out.println("This is III century");
+
+}else if(year>=300 && year<400){
+System.out.println("This is IV century");
+
+}else if(year>=400 && year<500){
+System.out.println("This is V century");
+
+}else if(year>=500 && year<600){
+System.out.println("This is VI century");
+
+}else if(year>=600 && year<700){
+System.out.println("This is VII century");
+
+}else if(year>=700 && year<800){
+System.out.println("This is VIII century");
+
+}else if(year>=800 && year<900){
+System.out.println("This is IX century");
+
+}else if(year>=900 && year<1000){
+System.out.println("This is X century");
+
+}else if(year>=1000 && year<1100){
+System.out.println("This is XI century");
+
+}else if(year>=1100 && year<1200){
+System.out.println("This is XII century");
+
+}else if(year>=1200 && year<1300){
+System.out.println("This is XIII century");
+
+}else if(year>=1300 && year<1400){
+System.out.println("This is XIV century");
+
+}else if(year>=1400 && year<1500){
+System.out.println("This is XV century");
+
+}else if(year>=1500 && year<1600){
+System.out.println("This is XVI century");
+
+}else if(year>=1600 && year<1700){
+System.out.println("This is XVII century");
+
+}else if(year>=1700 && year<1800){
+System.out.println("This is XVIII century");
+
+}else if(year>=1800 && year<1900){
+System.out.println("This is XIX century");
+
+}else if(year>=1900 && year<2000){
+System.out.println("This is XX century");
+
+}else if(year>=2000 && year<2100){
+System.out.println("This is XXI century");
+}
```

Comment:
technically it's correct, but there is a math expression to achieve result:
int century = (year / 100) + 1;